tools/manifest/item.py

```diff
@@ -84,7 +84,15 @@ def id(self):
 
     @property
     def url(self):
-        return urljoin(self.url_base, self._url)
+        # we can outperform urljoin, because we know we just have path relative URLs
+        if self._url.startswith("/"):
+            # TODO: MANIFEST6
+            # this is actually a bug in older generated manifests
+            return self._url
```

Comment:
Also is this actually correct if the `url_base` isn't `/`?

Reply:
Yes, given resolving `/foo/bar` against `/_mozilla` is `/foo/bar`. Really the problem is the old manifests are broken…